api/scpca_portal/test/management/commands/test_create_portal_metadata.py

```diff
@@ -51,19 +54,38 @@ def test_create_portal_metadata(self):
         self.load_test_data()
         self.processor.create_portal_metadata(clean_up_output_data=False)
 
-        with ZipFile(common.OUTPUT_PORTAL_METADATA_ZIP_FILE_PATH) as zip:
+        with ZipFile(LOCAL_ZIP_FILE_PATH) as zip:
```

Comment:
```suggestion
        zip_file_path = ComputedFile.get_local_file_path(common.GENERATED_PORTAL_METADATA_DOWNLOAD_CONFIG)
        with ZipFile(zip_file_path) as zip:
```

Reply:
This would be the preferred way to handle this for now. In the future we should persist the download_config to the computed file instance so we can query the computed file and then grab the local path as an attribute.